docs/release_notes.md

```diff
@@ -3,7 +3,8 @@
 ## Release v1.1.0
 
 ### Notable changes
-
+- Certificate rotation now ignores monotonic clock readings when checking expiration
```

Comment:
Should we add a separate release note for patch releases?

Reply:
Like a different section from notable changes or like a header under notable changes?
I added the previous patch release change under Notable changes though, so then maybe we'd want to implement this for releases going forward?

Reply:
You should have a separate note for each release, including patch releases. This file typically is used to populate the final release notes. Regardless, this should accurately convey the information for releases and not update the release notes of an already released version.

Reply:
Added sections for v1.1.2 and v1.1.3 and included the notable changes for each